src/backend/templates/views/markdown/privacy.md

```diff
@@ -37,16 +27,51 @@ When you create a new account, you agree to our collection of this User Personal
  - We will use your email address to communicate with you about important updates to our service.
  - We use your User Personal Information for internal purposes, such as moderation, maintaining logs for security purposes and for legal documentation.
+   - We retain certain technical identifiers (such as IP addresses and device/browser fingerprints) and 3rd party identifiers (such as SteamID or gog.com ID) where necessary to detect repeat abuse, prevent ban evasion, or investigate serious violations of our Terms of Service. This is based on our legitimate interest in maintaining the integrity and security of our community. These identifiers are not used for profiling, advertising, or user behavior analysis unrelated to security or moderation.
+   - We retain 3rd party identifiers of licensing providers (such as SteamID or gog.com ID), to legally document game license ownership of users when downloading game assets from our services.
  - If we need to use your User Personal Information for any reasons not listed here, we will ask for your permission before doing so.
 
 ## Our legal basis for processing information
 
 Under the GDPR, we’re required to notify you about the legal basis on which we process User Personal Information.
 
 When you create an account, we require your user name and email address to enter into the [Terms of Service](https://www.faforever.com/tos) agreement with us, and we use this data as the basis of that contract. We also process your email address and user name on other legal bases.
-The remainder of information processing we do is necessary for the purposes of our legitimate interests. For example, for security purposes, we keep logs of hardware information unique to your machine, in order to identify malicious users trying to spoof their identify.
+The remainder of information processing we do is necessary for the purposes of our legitimate interests. For example, for security purposes, we keep logs of hardware information unique to your machine, in order to identify malicious users trying to spoof their identity.
 If you would like to request erasure of your data, please email us at [admin@faforever.com](mailto:admin@faforever.com).
 
+
+## Data Controller
+
+```
+Forged Alliance Forever
+Kolt Østervej 58
+8361 Hasselager
+Denmark
+Email: admin@faforever.com
+```
+
+## Data Retention
+
+We retain personal data only as long as necessary to fulfill the purposes described or to comply with legal obligations. Specifically:
+- Account and profile data are retained while your account is active or until you request deletion.
+- Technical identifiers are stored up to 5 years after account termination
+- Game license ownership related data are stored up to 5 years after account termination
+- Log files and connection data (including IP addresses) are stored for up to 2 years.
+- Backups of data are kept for up to 2 years before being deleted or anonymized.
```

Comment:
Is the deletion of this data after the time expires automated?

Reply:
Partially. Some we do manually.